pytorch_toolkit/object_detection/custom-detection/problem.yml

```diff
@@ -0,0 +1,9 @@
+description:  >-
+  Custom object detection is a computer technology being used in a variety of applications
+  that identifies human in digital images. You need to define your set of detection classes.
+favorite: false
+imagesUrls:
+  - some_picture_url
+subtitle: Detection custom objects in digital images
+title: Custom Detection
+type: generic
```

Comment:
It is still not clear what `Custom Detection` is. Adding a readme is a good idea. Even if this is just a project template.

Reply:
added